### ISSUE TITLE
Very odd crash, no clue what's causing it

### ISSUE THREAD

**Issue by vizthex123** [Asek3/Rubidium#142]
Sorry if [this log](https://gist.github.com/vizthex123/3f85f010accb7c02aa989c12739ef985) doesn't help, but I found that removing the mod fixes the crash.

Like I said, I got no clue wtf the problem is, but hopefully you're able to fix it or at least warn people on the mod page.

**Comment by lSherozl** [comment]
Don't know whats causing your crash, but this mod is incompatible with tinkers construct, going anywhere near a slime island will crash your game

**Comment by vizthex123** [comment]
> Don't know whats causing your crash, but this mod is incompatible with tinkers construct, going anywhere near a slime island will crash your game

Yeah, mine crashed on load. I removed the mod and it works fine now, but figured I should report the issue just in case.

**Comment by ChiefArug** [comment]
The launchers log probably has a more helpful error, you can get it if you are using the Curseforge app (which you appear to be doing) by following these steps:
> 1. Right click a profile.
> 2. Select Open Folder
> 3. Go back two folders.
> 4. Open the Install folder.
> 5. launcher_log.txt is the file you are looking for, drag and drop it into github

**Comment by aethaerya** [comment]
> > Don't know whats causing your crash, but this mod is incompatible with tinkers construct, going anywhere near a slime island will crash your game
> 
> Yeah, mine crashed on load. I removed the mod and it works fine now, but figured I should report the issue just in case.

For the record, this mod doesn't cause crashes when with Tinkers' Construct, it crashes when used with an unknown third, possibly fourth mod upon nearing a slime island. The crash that lSherozl is experiencing is in a modpack with more than 460+ mods and the proper identification procedure was not followed.
When testing Rubidium + Oculus + Tinkers' Construct + Mantle and **ONLY** those mods, it cannot be reproduced, so one cannot say that this mod is incompatible with Tinkers'. Moreover, the crash that lSherozl is experiencing is on 1.16.5, which is irrelevant, as this issue is for 1.18.2.

You can view the details of lSherozl's specific issue here: https://github.com/Asek3/Rubidium/issues/140

**Comment by vizthex123** [comment]
> The launchers log probably has a more helpful error, you can get it if you are using the Curseforge app (which you appear to be doing) by following these steps:
> 
> > 1. Right click a profile.
> > 2. Select Open Folder
> > 3. Go back two folders.
> > 4. Open the Install folder.
> > 5. launcher_log.txt is the file you are looking for, drag and drop it into github

That got overridden awhile back (since I managed to fix my issue), so it's not going to be helpful. I suppose I'll keep it in mind though.
